seed/php-sdk/any-auth/src/Auth/AuthClient.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Seed\Auth;
+
+use Seed\Core\RawClient;
+
+class AuthClient
+{
+    /**
+     * @var RawClient $client
+     */
+    private RawClient $client;
+
+    /**
+     * @param RawClient $client
+     */
+    public function __construct(
+        RawClient $client,
```

Comment:
any reason not to do constructor property promotion for these? (can follow up with it)

Reply:
I originally left it out since it's a `private` member, but no good reason - we can easily refactor this to use promotion.

I'll leave this as-is for now, but will revisit in a follow-up when we start implementing the methods!